cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -366,7 +370,8 @@ private ActionModellingResult compileAction(Map<String, Object> actionRawData) {
 
         String errorMessagePrefix = "Action syntax is illegal.\n";
         actionData.putAll(
-                transformersHandler.runTransformers(actionRawData, actionTransformers, errors, errorMessagePrefix));
+                transformersHandler.runTransformers(actionRawData, actionTransformers, errors, errorMessagePrefix,
+                        SensitivityLevel.ENCRYPTED));
```

Comment:
@Bonczidai the param is needed only for the InputsTransformer, we don't need to pass this param for the other transformers